modules/recommended-configurable-storage-technology.adoc

```diff
@@ -120,7 +120,6 @@ Application use cases vary from application to application, as described in the
 
 == Other specific application storage recommendations
 
-* {product-title} Internal `etcd`: For the best `etcd` reliability, the lowest consistent latency storage technology is preferable.
 * It is highly recommended that you use `etcd` with storage that handles serial writes (fsync) quickly, such as NVMe or SSD. Ceph, NFS, and spinning disks are not recommended.
```

Comment:
that bullet still mentions etcd, can we remove that one too? thanks!

Reply:
Will do!